Updating README with TinaCMS dev badge

PothieuG opened the pull request:
The goal of this PR is to integrate the Tina CMS Badge process into the README. The process will be as follows:
- Join the Discord server.
- Create a project with Tina.
- Record a video demonstrating completion.
- Validate the process by sending an email to badge@tina.io.

PothieuG committed:
Updating Readme with TinaCMS Dev Licence steps

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tinacms/tinacms?pullRequest=4612) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tinacms/tinacms?pullRequest=4612) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tinacms/tinacms?pullRequest=4612) it.</sub>

bradystroud commented:
I wonder if there is a better term than "licence" (might get confused with the licence to the product)

brookjeynes-ssw commented:
CC: @bradystroud @ncn-ssw 

Hey @PothieuG and @adamcogan ,
 
I have concerns about this idea.
 
Are these steps mandatory to complete before an open-source PR can be approved? I believe these steps will scare away more contributors than attract.
 
Step 1: I wouldn't make joining the Discord mandatory. Instead, I would mention this is where developer discussions takes place. This would help guide the contributor.
 
Step 2: It would be better to link off to a developer README.md instead, which would describe the process of how to test your changes locally on a TinaCMS project.
 
Step 3: Having to record a video unrelated to their issue and about their understanding of TinaCMS seems like a huge ask for someone and will result in this step either being skipped by most or cause a dev to not want to continue with the process further.
 
Steps 4-6: Making the contributor have to email/post the video and wait for their contributor license to be approved will lead to longer wait times and frustration. This will also consume SSW dev time.
 
These steps make contributing harder.

wicksipedia commented:
Agree with @brookjeynes-ssw - as is, this may alienate the community as it can be perceived as a requirement to contribute back to the project.

> Hi, I am trying out TinaCMS with {{ TECH eg. NextJS }}...

This could be a suggestion when people join the Discord community as a template to follow on `#introductions`

Steps 2-6 - feel more internal SSW than anything else (especially the language/terms). I'd expect to see that on SugarLearning and not a public project. 

Also, I'm not sure that a 'done video' is enough to gauge whether someone is ready to use Tina on a client project.

For a cert, it may make more sense that a developer follows a tutorial that gets them to setup Tina, deploy it, then add some functionality. Then they submit the URL to that project, which can be checked (automated) for expected functionality

JackDevAU reviewed (commented):
As per my conversation with @adamcogan @wicksipedia @bradystroud 

This needs some slight changes before going live. 

I'll make this a draft PR for the moment till they have been finalised.

PothieuG committed:
Merge branch 'tinacms:main' into main

PothieuG committed:
Moving and modifying the TinaCMS Dev Licence documentation

JackDevAU commented on README.md:
```diff
@@ -37,6 +37,9 @@ Visit [Tina's documentation](https://tina.io/docs/) to learn more.
 
 Visit the [GitHub Discussions](https://github.com/tinacms/tinacms/discussions) or our [Community Discord](https://discord.com/invite/zumN63Ybpf) to ask questions, or look us up on on Twitter at [@tinacms](https://twitter.com/tinacms).
 
+## 🧑‍🎓 Tina Creadly Badge 
```
```suggestion
## 🧑‍🎓 Earn a Tina Badge
```

JackDevAU commented on README.md:
```diff
@@ -37,6 +37,9 @@ Visit [Tina's documentation](https://tina.io/docs/) to learn more.
 
+## 🧑‍🎓 Tina Creadly Badge 
+
+If you want to go the extra mile and earn a Tina Dev Licence, follow this doc: [Earn a Tina Credly badge](./_docs/Dev-Licence.md)
```
```suggestion
[Earn a Tina Credly badge](./_docs/Dev-Licence.md)

Go the extra mile and earn a Tina Dev Licence 🦙
```

What about this? 😆

JackDevAU commented on _docs/Dev-Licence.md:
```diff
@@ -0,0 +1,24 @@
+    > After I have evaluated TinaCMS, I plan to use it in the {{ INDUSTRY }} industry.  
+    > Previous CMSs I have used include {{ FILL IN }}.
+
+2. Create an Tina instance using the default template on [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin).
```
```suggestion
2. Create a Tina instance using the default template on [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin).
```

PothieuG committed:
Updating Tina badge title in Readme.md

Co-authored-by: Jack Pettit [SSW] <57518417+JackDevAU@users.noreply.github.com>

PothieuG committed:
Applying Jack suggestions (Readme.md)

wicksipedia commented:
@PothieuG 
1. PR needs a description
2. Is the email address setup? employee responsibility for monitoring it?

PothieuG commented:
@wicksipedia 

> 1. PR needs a description

✅ Done, just added few seconds ago.

> 2. Is the email address setup? employee responsibility for monitoring it?

✅ Yes.

![image](https://github.com/user-attachments/assets/ef9a4245-75fe-4775-91df-58cf38855163)
**Figure:** Warwick email mentioning the setup of badge@tina.io email being up and running

![image](https://github.com/user-attachments/assets/2aa378b8-5b9c-4e6a-88d1-13a45379d4e5)
**Figure:** Tina badge validator responsabilities in the CRM's Employee responsabilities section

wicksipedia commented on README.md:
```diff
@@ -37,6 +37,9 @@ Visit [Tina's documentation](https://tina.io/docs/) to learn more.
 
+## 🧑‍🎓 Earn a Tina Badge
+
+Go the extra mile and earn a Tina Dev Licence 🦙: [Earn a Tina Credly badge](./_docs/Dev-Licence.md)
```
```suggestion
Go the extra mile and earn a Tina Dev Licence 🦙: [Earn a Tina Credly badge](./_docs/dev-licence.md)
```

wicksipedia committed:
Rename Dev-Licence.md to dev-licence.md

wicksipedia committed:
Update README.md

JackDevAU commented on _docs/dev-licence.md:
```diff
@@ -0,0 +1,24 @@
+
+3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works.
+
+    Here is a sample Done Video: https://www.youtube.com/watch?v=OhVYTOKCsWI  
```
Is this the video we want to use? Should we change this to a Tina related video? (Maybe someone doing this exact process?)

wicksipedia replied:
dropped it, didn't make sense in the context of the video we want
added a list of things for us to look out for instead

wicksipedia committed:
Update and rename dev-licence.md to dev-badge.md

badge is more friendly than license
also updated requirements to make it clear as to what is required

wicksipedia committed:
Update README.md

wicksipedia committed:
Merge branch 'main' into main

wicksipedia committed:
Update README.md

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+## 🧑‍🎓 TinaCMS Dev Badge
+
+We encourage all developers to get their “TinaCMS badge” - that is, to get it up and running and complete the basics before implementing TinaCMS on a real project. This is a must to prove that you can use TinaCMS!
```
```suggestion
We encourage all developers to earn their “TinaCMS badge” by getting TinaCMS up and running and complete the basics before implementing TinaCMS on a real project. This is essential to prove that you can effectively use TinaCMS!
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+
+3. Record a [Done Video](https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/) explaining how Tina works
+
+   What we are looking for:
```
```suggestion
   Here's what we're looking for:
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+2. Get TinaCMS running and deploy a site using [tina.io](https://tina.io) and [TinaCloud](https://app.tina.io/signin)
+   
+   Feel free to start with one of the starter templates
+   This can be a personal site, or one you did for work
```
```suggestion
   This can be a personal site or one you’ve built for work
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+   - Any feedback or suggestions you have from your initial tryout
+     Tip: Have each suggestion's discussion or issue open in a separate tab
+
+4. Post the vidoe to YouTube (or other video platform) - make sure that it is public
```
```suggestion
4. Post the video to YouTube (or another video platform) - make sure that it is public
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+4. Post the vidoe to YouTube (or other video platform) - make sure that it is public
+5. (Optional) Post your video on social media (e.g., LinkedIn, Discord, X...)
+
+6. Let us know about it! Email badge@tina.io
```
```suggestion
6. Let us know about it! Email us at badge@tina.io
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+
+   Include a link to:
+   - your video
+   - links to your posts
```
```suggestion
   - Your social media posts (if applicable)
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+6. Let us know about it! Email badge@tina.io
+
+   Include a link to:
+   - your video
```
```suggestion
   - Your video
```

JackDevAU commented on _docs/dev-badge.md:
```diff
@@ -0,0 +1,36 @@
+
+6. Let us know about it! Email badge@tina.io
+
+   Include a link to:
```
```suggestion
   Include links to:
```

JackDevAU committed:
Apply suggestions from code review

JackDevAU reviewed (approved):
🦙
